templates/login.html.twig

```diff
@@ -78,7 +78,7 @@
 
 
     <!-- Critical CSS -->
-    <link rel="stylesheet" href="../css/main.min.css" type='text/css'>
+    <link rel="stylesheet" href="../assets/styles/main.min.css" type='text/css'>
```

Comment:
We should probably be using encore for assets, not fixed paths like this. https://symfony.com/doc/current/frontend.html

Reply:
It can be used for all assets, not just JS.